### ISSUE TITLE
[Accessibility] Color Contrast is not respected.

### ISSUE THREAD

**Issue by andreacantone** [SmartReceipts/SmartReceiptsLibrary#253]
## Description

The contrast ratio between text and background should be at least 4.5 . Some elements don't respect the contrast ratio. 

## How to reproduce the error

1. Activate the app [Accessibility Scanner](https://play.google.com/store/apps/details?id=com.google.android.apps.accessibility.auditor&hl=it&gl=US)
2. Open the app [Smart Receipts](https://play.google.com/store/apps/details?id=wb.receipts&hl=it&gl=US)
3. Activate the registration from the Accessibility Scanner and use the Smart Receipts app.
4. After using the app, stop the recording.

## Screenshot

The results will show the errors found in the app. For example:

![IMG_20201119_095358](https://user-images.githubusercontent.com/25429624/99647168-b0b06580-2a51-11eb-8bb2-774703ebd3b5.png)
